lib/active_model/serializer.rb

```diff
@@ -21,7 +21,7 @@ class Serializer
     include Caching
     include Links
     include Type
-    require 'active_model/serializer/adapter'
+    require 'active_model_serializers/adapter'
```

Comment:
I think this should be removed now, along with the `adapter` method (which I guess we can keep around with a deprecation warnings)